### PR TITLE
adding hint that reading the user profile from a screen hook can help…

### PR DESCRIPTION
**Pull request by antoinerabanes**
… with personalisation. i.e. if profile contains data A, present screen X, otherwise present screen Y.

## Checklist

* [x] PR is scoped to one task
* [ ] Tests are included
* [x] Documentation is included

* [x] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: book/login/intro.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Login plugins are used in order to integrate the following capabilities:
 * User authentication: authenticate existing users with the backend
 * User authorization: after authenticating, make sure the user has a valid entitlement for an action
 * Offering various payment models: usually through in app purchases like subscriptions and one time purchases
-* Expose user profile within the app for various uses such as analytics
+* Expose user profile within the app for various uses such as analytics or personalisation
```
Comment (snooky23) [inline book/login/intro.md]: Fix typo - `personalisation ` should be `personalization`

Comment (snooky23) [reply book/login/intro.md]: Change the sentence to: `Expose user profile within the app for various uses such as analytics and personalization`.

**Review by snooky23 — CHANGES_REQUESTED** [review]
Please fix my comments and ping me for approval.

File: book/login/intro.md (hunk 1 above)
Comment (liviur) [reply book/login/intro.md]: @snooky23 - side note - this is not a type-o its the difference between "American" and "British" type spelling.
That said - we are keeping it American style in the portal so it should indeed be with a z

Comment (antoinerabanes) [reply book/login/intro.md]: :D

**Review by liviur — APPROVED** [review]

**Review by snooky23 — APPROVED** [review]

**Comment by lomudi** [comment]
@antoinerabanes you are planning to merge it?

**Pull request merged**